src/overlays/actors/ovl_Demo_Go/z_demo_go.c

```diff
@@ -200,21 +200,16 @@ UNK_TYPE DemoGo_FrameUpdateMatrix(DemoGo* this) {
     return SkelAnime_FrameUpdateMatrix(&this->skelAnime);
 }
 
-#ifdef NON_MATCHING
-// return value isn't produced in the same way
 s32 func_8097CDB0(DemoGo* this, GlobalContext* globalCtx, u16 csCmdNPCAction) {
     CutsceneContext* csCtx = &globalCtx->csCtx;
-    CsCmdActorAction* npcAction = csCtx->npcActions[func_8097C870(this)];
-    if (csCtx->state != 0) {
-        if (npcAction != NULL && npcAction->action == csCmdNPCAction) {
-            return 1;
-        }
+    s32 actionIdx = func_8097C870(this);
+
+    if ((csCtx->state != 0) && (csCtx->npcActions[actionIdx] != NULL) &&
+        (csCtx->npcActions[actionIdx]->action == csCmdNPCAction)) {
```

Comment:
common name for this temp is npcAction, works well imo. context that its from a cs command is obvious
I know its the name that was already there, just saying in general

Reply:
wait im confused, theres an npcAction temp and another one thats passed in?

Reply:
nvm im just seeing things. rename arg imo

Reply:
kk yeah I thought the arg name was strange but I left it as is